.github/workflows/land-blocking.yml

```diff
@@ -54,7 +55,7 @@ jobs:
         if: steps.check_ks.outputs.should_run == 'true'
         run: |
           set +e
-          commit_message=$(git log -1 --pretty=oneline)
+          commit_message=$(git log --pretty=oneline $MASTER_GIT_REV..$LIBRA_GIT_REV)
```

Comment:
I don't think you _need_ to rev-parse, you can probably do smth like `origin/master..$LIBRA_GIT_REV`, but up to you

Reply:
Leaving like this for now, but thanks for pointing that out. I think one thing in the future is we might want to check against `origin/testnet` for client-compatibility. e.g. because pylibra and other clients will only be known compatible with latest testnet version.